src/UniversalRewardsDistributor.sol

```diff
@@ -98,9 +97,9 @@ contract UniversalRewardsDistributor is IUniversalRewardsDistributor {
         root = pendingRoot.root;
         ipfsHash = pendingRoot.ipfsHash;
 
-        emit EventsLib.RootSet(pendingRoot.root, pendingRoot.ipfsHash);
+        _setPendingRoot(PendingRoot(0, 0, 0));
 
-        delete pendingRoot;
+        emit EventsLib.RootSet(root, ipfsHash);
```

Comment:
Why aren't we using `_setRoot` here?

Reply:
Done in another PR (#81), but I'll do it in this PR since the logic is moving